src/Exceptions/UnauthorizedException.php

```diff
@@ -52,6 +53,13 @@ public static function forRolesOrPermissions(array $rolesOrPermissions): self
         return $exception;
     }
 
+    public static function notUseHasRoles(Authorizable $user): self
```

Comment:
Do you mind renaming this to `missingTraitHasRoles()`, to improve readability?

One thing "different" about this method call is that it's an application programming error, unlike the others in this class which are more about actual authorization failures.
It's probably fine to leave it here, because it does specifically deal with why the application will fail to do authorizations with this package until the app error is fixed.

Reply:
> It's probably fine to leave it here

So, should I add an exception in larastan and close this PR?

Reply:
> should I add an exception in larastan and close this PR?

No, I think these are probably good changes to implement.